swedish-oidc-profile.md

```diff
@@ -461,31 +459,57 @@ following table.
 <a name="security-requirements"></a>
 ## 7. Security Requirements
 
-All transactions MUST be protected in transit by TLS (TODO: include reference).
+All transactions MUST be protected in transit by TLS as described in \[[NIST.800-52.Rev2](#nist800-52)\]. 
 
 All parties MUST conform to applicable recommendations in section 16, "Security Considerations" of \[[OAuth2.RFC6749](#oauth2-rfc6749)\] and those found in "OAuth 2.0 Threat Model and Security Considerations", \[[RFC6819](#rfc6819)\].
 
-> **To discuss:** Is this too vague?
-
 <a name="cryptographic-algorithms"></a>
 ### 7.1. Cryptographic Algorithms
 
 This section lists the requirements for crypto algorithm support for being compliant with this profile.
 
-For signature and encryption keys the following requirements apply:
+All entities compliant with this profile MUST follow the guidelines in \[[NIST.800-131A.Rev2](#nist800-131)\]
+regarding use of algorithms and key lengths<sup>1</sup>. Specifically, for signature and encryption keys the
+following requirements apply:
+
```

Comment:
General comment:

For this specification I believe that it makes sense to have different requirement levels for signing and verification. Alternatively if requirements are specified only for verifications, they could be upgraded.

It is reasonable to require that all implementations MUST be able to validate a signature based on the common algorithms RS256, RS384, RS512, ES256, ES384 and ES512.

In the end the most useful thing to do is to say MUST support validation based on all of these, MUST use one of them for signing and finally perhaps SHOULD use the most preferred one (RS256 and ES256)

Reply:
I think it will be too confusing to start writing about different usages. Better to keep it simple...

Reply:
Given that capabilities are provided in metadata, we can have the base profile as proposed. However. I do think we should consider making all these algorithms mandatory to support for practical use in Sweden Connect. I don't think this changed what is used in practice though.